rest.c

```diff
@@ -242,6 +364,9 @@ rest_api_search (const char *path, const char *if_none_match, const char *if_mod
     rc = apteryx_json_search (path, &data);
 
   exit:
+    if ((flags & FLAGS_RESTCONF))
```

Comment:
should not be restconf specific (that flag will be removed eventually) ... but should be "if (logging)"

Reply:
Done